server: TestClusterVersionUpgrade failed

server.TestClusterVersionUpgrade [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9642427?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9642427?buildTab=artifacts#/) on release-23.1 @ [58b140936197618750d984d6c483284abaecaba5](https://github.com/cockroachdb/cockroach/commits/58b140936197618750d984d6c483284abaecaba5):


```
=== RUN   TestClusterVersionUpgrade
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/5b2c9b3a394428c7572d34050aad8975/logTestClusterVersionUpgrade1255177517
    test_log_scope.go:79: use -show-logs to present logs inline
    version_cluster_test.go:264: condition failed to evaluate within 3m0s: cluster version is still 22.2-96, should be 22.2-98
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/5b2c9b3a394428c7572d34050aad8975/logTestClusterVersionUpgrade1255177517
--- FAIL: TestClusterVersionUpgrade (184.75s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101593 server: TestClusterVersionUpgrade failed [C-test-failure O-robot T-server-and-security branch-release-23.1.0]
</p>
</details>
/cc @cockroachdb/server
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestClusterVersionUpgrade.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-27074

server.TestClusterVersionUpgrade [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9662902?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9662902?buildTab=artifacts#/) on release-23.1 @ [12607684868d04f957b3385f142e4e8b9b55c799](https://github.com/cockroachdb/cockroach/commits/12607684868d04f957b3385f142e4e8b9b55c799):


```
=== RUN   TestClusterVersionUpgrade
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/5b2c9b3a394428c7572d34050aad8975/logTestClusterVersionUpgrade3168060806
    test_log_scope.go:79: use -show-logs to present logs inline
    version_cluster_test.go:264: condition failed to evaluate within 3m0s: cluster version is still 22.2-94, should be 22.2-98
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/5b2c9b3a394428c7572d34050aad8975/logTestClusterVersionUpgrade3168060806
--- FAIL: TestClusterVersionUpgrade (184.91s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101593 server: TestClusterVersionUpgrade failed [C-test-failure O-robot T-server-and-security branch-release-23.1.0]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestClusterVersionUpgrade.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

server.TestClusterVersionUpgrade [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9920396?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9920396?buildTab=artifacts#/) on release-23.1 @ [c44b4bd3125b406d176d1ccb6dbd7f4a4bad0cc9](https://github.com/cockroachdb/cockroach/commits/c44b4bd3125b406d176d1ccb6dbd7f4a4bad0cc9):


```
=== RUN   TestClusterVersionUpgrade
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/5b2c9b3a394428c7572d34050aad8975/logTestClusterVersionUpgrade3787813464
    test_log_scope.go:79: use -show-logs to present logs inline
    version_cluster_test.go:264: condition failed to evaluate within 3m0s: cluster version is still 22.2-94, should be 23.1
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/5b2c9b3a394428c7572d34050aad8975/logTestClusterVersionUpgrade3787813464
--- FAIL: TestClusterVersionUpgrade (184.83s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101593 server: TestClusterVersionUpgrade failed [C-test-failure O-robot T-server-and-security branch-release-23.1.0]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestClusterVersionUpgrade.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

server.TestClusterVersionUpgrade [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/10450206?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/10450206?buildTab=artifacts#/) on release-23.1 @ [b8c62562e3b5b26acb75a195754c1945009a58fb](https://github.com/cockroachdb/cockroach/commits/b8c62562e3b5b26acb75a195754c1945009a58fb):


```
=== RUN   TestClusterVersionUpgrade
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/5b2c9b3a394428c7572d34050aad8975/logTestClusterVersionUpgrade2371421971
    test_log_scope.go:79: use -show-logs to present logs inline
    version_cluster_test.go:264: condition failed to evaluate within 3m0s: cluster version is still 22.2-92, should be 23.1
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/5b2c9b3a394428c7572d34050aad8975/logTestClusterVersionUpgrade2371421971
--- FAIL: TestClusterVersionUpgrade (184.25s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101593 server: TestClusterVersionUpgrade failed [C-test-failure O-robot T-dev-inf todo-deprecate.branch-release-23.1.0]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestClusterVersionUpgrade.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

server.TestClusterVersionUpgrade [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/10659375?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/10659375?buildTab=artifacts#/) on release-23.1 @ [dbf6ce8438742a73b9d17391c1ef868a4e7a9569](https://github.com/cockroachdb/cockroach/commits/dbf6ce8438742a73b9d17391c1ef868a4e7a9569):


```
=== RUN   TestClusterVersionUpgrade
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/5b2c9b3a394428c7572d34050aad8975/logTestClusterVersionUpgrade112828921
    test_log_scope.go:79: use -show-logs to present logs inline
    version_cluster_test.go:264: condition failed to evaluate within 3m0s: cluster version is still 22.2-94, should be 23.1
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/5b2c9b3a394428c7572d34050aad8975/logTestClusterVersionUpgrade112828921
--- FAIL: TestClusterVersionUpgrade (184.56s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101593 server: TestClusterVersionUpgrade failed [C-test-failure O-robot T-dev-inf todo-deprecate.branch-release-23.1.0]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestClusterVersionUpgrade.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/server/version_cluster_test.go
+++ b/pkg/server/version_cluster_test.go
@@ -208,6 +208,7 @@ func TestClusterVersionUpgrade(t *testing.T) {
 
 	skip.UnderShort(t, "test takes minutes")
 	skip.UnderRace(t, "takes >5mn under race")
+	skip.UnderStress(t, "takes >3mn under stress")
 
 	ctx := context.Background()
 

